src/app/services/environments.service.ts

```diff
@@ -377,6 +377,36 @@ export class EnvironmentsService {
     }
   }
 
+  /**
+   * Start / stop all environments
+   */
+  public toggleAllEnvironments() {
+    const environments = this.store.get('environments');
+    const environmentsStatus = this.store.get('environmentsStatus');
+
+    // check if environments should be started or stopped. If at least one env is turned off, we'll turn all on
+    const shouldStart = Object.keys(environmentsStatus)
+      .map(uuid => environmentsStatus[uuid].running)
+      .some(status => !status);
```

Comment:
You can shorten this with only `some`
```suggestion
    const shouldStart = Object.keys(environmentsStatus)
      .some((uuid) => !environmentsStatus[uuid].running);
```